CHANGELOG.asc

```diff
@@ -6,4 +6,35 @@ Release Notes
 Version 0.1.0 (TBD)
 ~~~~~~~~~~~~~~~~~~~
 
-Changelog to be added once released.
+[source, m2]
+<dependency>
+   <groupId>org.janusgraph</groupId>
+   <artifactId>janusgraph-core</artifactId>
+   <version>0.1.0</version>
+</dependency>
+
+*Tested Compatibility:*
+
+* Cassandra 2.1.9
+* HBase 0.98.8-hadoop2, 1.0.3, 1.1.8, 1.2.4
+* BerkeleyJE 7.3.7
+* Elasticsearch 1.5.1
+* Lucene 4.10.4
+* Solr 5.2.1
+* TinkerPop 3.2.3
+* Java 1.8
+
+*Features added since version Titan 1.0.0:*
+
+* TinkerPop 3.2.3 compatibility
+** Includes update to Spark 1.6.1
+* Query optimizations: JanusGraphStep folds in HasId and HasContainers can be folded in even mid-traversal
```

Comment:
@twilmes Can you review this line? I want to summarize the updates in [49e3bde](https://github.com/JanusGraph/janusgraph/commit/49e3bde74519d797b92e22569027ce332917e0f9) and [c9ebb1c](https://github.com/thinkaurelius/titan/commit/c9ebb1c676110cb6ebfa50e4d412ebea9aac54a4).